CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. Feature request issues should answer these questions: 
+
+```
+- What would you like implemented? What do you want that Twilight lacks?
+- What use case does this request address? This should be more than a single project's use case.
+- Are you willing to help towards contributing this feature?
+- Is there any other information that we should know?
```

Comment:
I feel like this is duplication and would just say to use the according templates